build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml

```diff
@@ -13,15 +13,19 @@ parameters:
   # if 'useBuildOutputFromBuildId' is set, we will default to using a build from this pipeline:
   useBuildOutputFromPipeline: $(System.DefinitionId)
   matrix: 
+    Debug_x86:
+      buildPlatform: 'x86'
```

Comment:
>x86 [](start = 22, length = 3)

should we do x64 debug for one of these flavors?

Reply:
That would require adding another build flavor to the mix. We need to be judicious in our use of build resources. I think the current set of build configurations should give us a reasonable level of coverage.